examples/nlp/gpt/train_gpt_sft.py

```diff
@@ -119,15 +126,34 @@ def main(cfg) -> None:
     with open_dict(cfg):
         cfg.model.precision = cfg.trainer.precision
 
-    ptl_model = load_from_nemo(
+    assert cfg.model.peft.peft_scheme in ["lora", "none"], "Only support LoRA or Full finetuning"
```

Comment:
maybe the `initialize_peft` function should take in a peft config

Reply:
09ad8c61852debbbd2e713396f4b99f1441b52f5